README.md

```diff
@@ -32,8 +32,12 @@ Alpine Linux and is using Docker.
 The installation of this add-on is pretty straightforward and not different in
 comparison to installing any other Hass.io add-on.
 
-**NOTE**: Your Hass.io machine MUST HAVE a static IP for this add-on to function
+**NOTE 1**: Your Hass.io machine MUST HAVE a static IP (not DHCP reserved, but static) for this add-on to function
 properly/fully.
+**NOTE 2**: Since Pi-hole will act as DNS server on your whole local network, you will need to install the `DNSMasq` add-on to make sure hass-io itself can still access the internet using public DNS servers (prevent `NewConnectionError` errors)
```

Comment:
This is incorrect. There are multiple solutions possible here. In general you have to make sure it doesn't become a client to itself.

Reply:
In the rewrites it now talks about loops for some odd reason.
As long as the Pihole DNS setting is externally AND there is no forced DNS redirect on the router, it wouldn't run into loops.